src/components/styled-list/components/StyledListItem.vue

```diff
@@ -19,7 +19,7 @@
 
                 <VsHeading
                     :level="props.headingLevel"
-                    heading-style="heading-l"
+                    heading-style="heading-3"
```

Comment:
Isn't heading-l correct here in the new system? The level prop takes a number 1-6 but the style prop takes display-/heading- and the letter